pwaCustomButton.js

```diff
@@ -0,0 +1,20 @@
+if (typeof window !== 'undefined') {
+  window.addEventListener('beforeinstallprompt', (e) => {
+    const pwaBtn = document.querySelector('#pwa-button');
+
+    // Prevent Chrome 67 and earlier from automatically showing the prompt
+    e.preventDefault();
+    deferredPrompt = e;
+
+    // Display A2HS button if app not installed
+    pwaBtn.style.display = 'block';
+
+    pwaBtn.addEventListener('click', (e) => {
+      // hide our user interface that shows our A2HS button
+      pwaBtn.style.display = 'none';
+
+      // Show the prompt
+      deferredPrompt.prompt().catch(() => {});
```

Comment:
Should we at least log something the console for debugging?

Reply:
Added an `console.error()` to display error occurs in `prompt()`